security/AccessControl.rst

```diff
@@ -18,11 +18,22 @@ that a user or group may perform, whether to invoke a function or access
 data, are organized into a hierarchy, and privileges may be granted at
 any level.
 
+Users and Groups can be updated using the Users and Groups Kylo admin pages:
```

Comment:
Actually, it is not strictly required that you use auth-kylo when configuring AD as long as you turn on group loading when using AD.  You are required, though, to have those AD groups added to Kylo with their same system names set exactly as they appear in AD.  You will also want to assign the appropriate permissions to those groups in Kylo as well.

The reason why you might want to combine auth-ad with auth-kylo is if the groups in AD are not applicable to you access control needs in Kylo, or if you would want to restrict the allowed Kylo users to a smaller set than can be authenticated through AD.  In these cases you would want to then add users to Kylo as well as the groups.